integration_tests/coverage_report/not-required-in-test-suite.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+throw new Error(`
+  this error should not be a problem because
+  this file is never required or executed
+`);
+
+module.exports = function(j, d) { // eslint-disable-line
```

Comment:
oh yeah that makes sense. Can you be specific and use `// eslint-disable <unreachable-code-plugin-name>` instead? Otherwise other people will wonder the same.

(YES, I'm all for expressive comments).

Also, please add the comment above the line, not next to it. You may have convinced me to add more comments, but we are not going to put them at the end of a line! I want them on their own lines in all of their glory.